app/views/new_contact_message_mailer/contact_message_notification.html.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1 style="color:#363691">A visitor to Any Rat Rescue has sent you a message.</h1>
+    <div>
+      <table style="border-collapse: collapse;">
+        <% ContactMessage::FORM_FIELDS.each do |field| %>
```

Comment:
@kenyonj I misunderstood. I thought you meant to move the constant over to be defined in the helper rather than the model. 
I think I would prefer to remove the helper and have the logic here. It's not particularly logic heavy or cumbersome. I think having another file for calling the constant would be more work than is necessary.